services/core/SQLiteTaggingService/sqlite/resources/categories.csv

```diff
@@ -0,0 +1,27 @@
+name,description
```

Comment:
I put it in two location since in theory they could be customized - adding new tags, categories etc. Also so that I could use resource_string method from package_resources